src/views/RecIM/components/List/index.jsx

```diff
@@ -58,15 +58,16 @@ const ParticipantList = ({
 const MatchList = ({ matches, activityID }) => {
   if (!matches?.length || !matches[0]) return <Typography>No matches to show.</Typography>;
   let content = matches.map((match) => (
-    <MatchListing key={match.ID} match={match} activityID={activityID} />
+    <MatchListing key={match?.ID} match={match} activityID={activityID} />
+    // I have no idea why, but on ladder matches, match can't be found
+    // despite it showing up on the debugger.
   ));
 
   return <List dense>{content}</List>;
 };
 // setTargetTeamID is used for edit Match teams
-const TeamList = ({ teams, match, invite, setInvites, setTargetTeamID }) => {
+const TeamList = ({ teams, match, series, invite, setInvites, setTargetTeamID }) => {
   const navigate = useNavigate();
-  if (!teams?.length && !match) return <Typography>No teams to show.</Typography>;
```

Comment:
I'm so confused on what you want. You requested that `!series` be removed because the logic fails. But then you want !match to stay in there? I think there's some logic flow that might not make sense to you at this point because if you're concerned about the fact that this is a good check to simply exist, we should be putting `!series` in there too. 

Also I think you believe that I tossed that `!series` conditional inside of this if statement without thinking and that I'm doing this simply because it happens everywhere else. This is not the case, I add that `!series` conditional because otherwise, teams in the series tab drop down **DO NOT RENDER** because I do not propagate the team or match field.  Once again, I beg the question, what do you mean that the boolean logic does not make sense and that I need an "or"

Just a reminder:
Home page -> TeamList(**team**, _no match_, _no series_)
MatchStatsEdit -> TeamList(_no team_, **match**, _no series_)
Activity page -> TeamList(_no team_, _no match_, **series**)

So, the question stands, **do you want this check in there or not**. If you want the check, I'm putting `!series` in there as well. If you want to sanitize BEFORE we put anything into TeamList then we should remove the check.